### DISCUSSION TITLE
Welcome to pysteamauth Discussions!

### DISCUSSION THREAD

**Discussion by sometastycake** [sometastycake/pysteamauth#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RandomNick2** [comment]
Как сделать подобные protobuf message для c#. Ты как то их генерируешь? Я хочу написать подобный проект но на c#
